odl/space/npy_ntuples.py

```diff
@@ -1237,22 +1235,27 @@ def conj(self, out=None):
 
         Examples
         --------
-        >>> x = NumpyFn(3, dtype=complex).element([5+1j, 3, 2-2j])
-        >>> y = x.conj(); print(y)
-        [(5-1j), (3-0j), (2+2j)]
+        Default usage:
+
+        >>> x = odl.cn(3).element([5+1j, 3, 2-2j])
+        >>> y = x.conj()
```

Comment:
Why not simply `x.conj()`, is `y` needed?

Reply:
No reason